src/python/grpcio/grpc/_cython/_cygrpc/call.pyx.pxi

```diff
@@ -106,6 +107,8 @@ cdef class Call:
   def __dealloc__(self):
     if self.c_call != NULL:
       grpc_call_destroy(self.c_call)
+    grpc_shutdown()
```

Comment:
I suspect the Core authors are going to want to update that documentation to be more in line with the conversation happening [here](https://github.com/grpc/grpc/issues/7478)?